packages/App/src/components/Input.tsx

```diff
@@ -0,0 +1,55 @@
+import React, {ChangeEvent, FC} from 'react';
+import {View, StyleSheet, TextInput, Text, TextInputProps} from 'react-native';
+
+interface Props extends TextInputProps {
+  label: string;
+  change: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const Input: FC<Props> = ({label, change, ...textInputProps}) => {
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
We are not working with HTML here. When you say Typescript that the argument is what it's not, it says you can access values that do not exist on your actual argument. This may lead to crashes in runtime